lib/virtmanager.pm

```diff
@@ -584,4 +584,27 @@ sub create_guest {
     assert_and_click 'virtman_guest_begin_install';
 }
 
+# Expecting to see the guest screen
+sub detect_login_screen {
+    my $timeout = shift // 5;
+    if (!check_screen 'virt-manager_login-screen', $timeout) {
+        wait_still_screen 3;
+        mouse_set(30, 200);
+        mouse_click();
+        save_screenshot();
+        hold_key "ctrl-alt";
```

Comment:
`hold_key` followed by `release_key` - what does it do?

Reply:
It escapes from the guest KVM, e.g. it releases the keyboard and the mouse pointer. Is it wrong in your opinion @kravciak?

Reply:
ah, no - now it's clear, maybe some comments would be useful :)